modules/book.js

```diff
@@ -0,0 +1,34 @@
+export default class Books {
+  constructor() {
+    this.bookList = [];
+  }
+
+  static fetchBooks() {
+    const getBooks = localStorage.getItem('books');
+    return getBooks ? JSON.parse(getBooks) : [];
+  }
+
+  static updateBooks(books) {
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static getBooksList() {
+    if (Books.fetchBooks()) {
+      this.bookList = Books.fetchBooks();
+    }
+  }
+
+  addBook(book) {
+    this.bookList = Books.fetchBooks();
+    this.bookList.push(book);
+    Books.updateBooks(this.bookList);
+  }
+
+  remove(e) {
+    const id = parseInt(e.target.id, 10);
+    this.bookList = Books.fetchBooks();
+    this.bookList = this.bookList.filter((item) => item.id !== id);
+    e.target.parentElement.remove();
+    Books.updateBooks(this.bookList);
+  }
```

Comment:
- [ ] **Good job on dividing your codes into modules 👏 👏** However for the sake of practice, I think it would be better to kindly divide your code into more modules and create a separate file for each module in the modules directory.

   Remember to Import added modules too into the index.js file. _(Tip: the more modules the better!)_ 👍 👍